rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/ArtifactDownloaderImpl.java

```diff
@@ -28,13 +27,7 @@ public ArtifactDownloaderImpl(ClouddriverService clouddriverService) {
 
   public InputStream downloadArtifact(Artifact artifact) throws IOException {
     Response response =
-        retrySupport.retry(
-            () ->
-                AuthenticatedRequest.allowAnonymous(
```

Comment:
What's the reasoning for removing anonymous calls? Does X-SPINNAKER-USER not just get set to Anonymous?

Reply:
The reasoning for adding allowAnonymous (which happened in https://github.com/spinnaker/rosco/pull/748) was to quiet down a bunch of log noise.  But now that SpinnakerRequestInterceptor is in place, there isn't any log noise to quiet down.  X-SPINNAKER-USER in the request to clouddriver is the same as it is in the request to rosco.